server/main_test.go

```diff
@@ -34,6 +35,24 @@ func TestMain(m *testing.M) {
 	os.Exit(status)
 }
 
+type AssignedPrompt struct {
```

Comment:
@mick-warehime this is what I was referring to as deserializing JSON into a struct for these testst

Reply:
gotcha - my thoughts are that by testing exact equality of the response means these test require more maintenance. every time we add a new field to the game state we have to add it to these tests even if they aren't testing that behavior - this isn't a huge concern but something worth considering.

Reply:
I see... What if we have a different model for the response (that we can sub out for a proto model in the future) and have each state own populating that object through `gameStatusForPlayer` kind of method? 

That way we don't have to send back all model fields to the user and it'll be easier for each state to populate the response model with the relevant data. Unused fields would then just have the default values and the tests wouldn't break (kind of the way I don't have to set points / host /prompts on any of the expected responses that don't use them)